src/discoveryDetailsContentGroup.js

```diff
@@ -27,4 +27,4 @@ if (metaTag) {
 
 if (typeof module !== 'undefined' && typeof module.exports !== 'undefined') {
```

Comment:
See previous comment - if we change our approach we won't need to perform this kind of test)